frontend/helpers/tillDuration.ts

```diff
@@ -20,17 +20,26 @@ export const formatSeconds = function (seconds: number): string {
     return formatDuration(duration);
 };
 
-export const formatInterval = function (startDate: Date, endDate: Date, isCountUp: boolean = false): string {
-    if (startDate < endDate || isCountUp) {
-        // switch startDate and endDate to act as a count-up timer
+export const formatInterval = function (startDate: Date, endDate: Date): string {
+    if (startDate < endDate) {
         const duration = intervalToDuration({
-            start: isCountUp ? endDate : startDate,
-            end: isCountUp ? startDate : endDate,
+            start: startDate,
+            end: endDate,
         });
         return formatDuration(duration);
     } else {
-        return formatDistance(endDate, startDate, {
-            addSuffix: true,
+        // check if difference between start and end dates is more than one month
+        if (!isSameMonth(startDate, endDate)) {
```

Comment:
That's not what you're checking here, [`isSameMonth` checks if two dates are in the same calendar month](https://date-fns.org/v2.29.3/docs/isSameMonth)

Instead, you can use [differenceInDays](https://date-fns.org/v2.29.3/docs/differenceInDays) or [differenceInMonths](https://date-fns.org/v2.29.3/docs/differenceInMonths)